sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/ShowTablesExec.scala

```diff
@@ -53,4 +80,103 @@ case class ShowTablesExec(
       case _ => false
     }
   }
+
+  private def extendedTable(
```

Comment:
the name looks weird, how about `getTableDetails`?

Reply:
Okay